src/auxinfo/participant.rs

```diff
@@ -693,6 +700,9 @@ mod tests {
         let outputs: Vec<_> = outputs.into_iter().flatten().collect();
         assert!(outputs.len() == QUORUM_SIZE);
 
+        let mut participant_ids = quorum[0].all_participants();
+        participant_ids.sort();
```

Comment:
nit: I would put the sort into the `fill_context` function if you can. That way if it gets reused in other tests we don't have to do this prep work every time.